Makefile

```diff
@@ -296,6 +296,12 @@ run_full_flow_with_ipv6:
 # Day 2 #
 #########
 
+install_day1_and_day2_cloud: start_load_balancer
+	TEST_TEARDOWN=no TEST=./discovery-infra/tests/test_targets.py TEST_FUNC=test_target_install_with_deploy_day2_nodes_cloud $(MAKE) test
+
+deploy_day2_nodes: start_load_balancer
+	TEST_TEARDOWN=no TEST=./discovery-infra/tests/test_targets.py TEST_FUNC=test_target_deploy_day2_nodes_cloud $(MAKE) test
+
```

Comment:
we're going lately for total deprecation of specialized makefile targets
meaning just using ``make test`` or ``make test_parallel`` with their matching arguments